src/client/Microsoft.Identity.Client/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -48,6 +54,12 @@ internal MsalAccessTokenCacheItem(
             RawClientInfo = response.ClientInfo;
             HomeAccountId = homeAccountId;
             OboCacheKey = oboCacheKey;
+
+            if (cacheKeyComponents != null && cacheKeyComponents.Any())

```

Comment:
can this go into a helper? `InitializeAdditionalCacheKeyComponents`

Reply:
Yeah